upb/msg.c

```diff
@@ -342,8 +340,8 @@ static upb_msglayout *upb_msglayout_new(const upb_msgdef *m) {
   }
 
   /* Size of the entire structure should be a multiple of its greatest
-   * alignment. */
-  l->data.size = align_up(l->data.size, l->data.align);
+   * alignment.  TODO: track overall alignment for real? */
+  l->data.size = align_up(l->data.size, 8);
```

Comment:
Ah, that is something slightly different. The runtime requires that a msginit has its size aligned to `sizeof(void*)` because it will add internal members that are `void*`.  However the message itself might require even greater alignment if it has int64/uint64 members.